### ISSUE TITLE
Bug -- Unable to use Apps

### ISSUE THREAD

**Issue by cyberczar** [pixelfed/pixelfed#2654]
**Note:** `.env` already has `OAUTH_ENABLED=true`, and has since day one.

Using an app such as FediLab or PixelDroid to sign-in to the instance results in the following in nginx `access.log`:

```
[redacted] - - [13/Feb/2021:14:20:58 +0000] "POST /api/v1/apps HTTP/1.1" 200 186 "-" "Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0"
[redacted] - - [13/Feb/2021:14:21:00 +0000] "GET /.well-known/nodeinfo HTTP/1.1" 200 138 "-" "Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0"
[redacted] - - [13/Feb/2021:14:21:00 +0000] "GET /api/nodeinfo/2.0.json HTTP/1.1" 200 1317 "-" "Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0"
[redacted] - - [13/Feb/2021:14:21:00 +0000] "POST /api/v1/apps HTTP/1.1" 200 188 "-" "Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0"
[redacted] - - [13/Feb/2021:14:21:00 +0000] "GET /oauth/authorize?client_id=17&redirect_uri=mastalab%3A%2F%2Fbacktomastalab&response_type=code&scope=read%20write%20follow HTTP/2.0" 500 4932 "-" "Mozilla/5.0 (Windows NT 6.1; rv:60.0) Gecko/20100101 Firefox/60.0"
```

And a PixelFed page that says "Something went wrong".

**Comment by stelzch** [comment]
I had a similar problem. After peeking into storage/logs/laravel.log, I saw the following: 

> production.ERROR: Key path "file:///var/www/pixelfed/storage/oauth-private.key" does not exist or is not readable

Which after a quick web search lead me to [this StackOverflow answer](https://stackoverflow.com/a/42796964).

TL;DR:
You need to run `php artisan passport:install`

**Comment by cyberczar** [comment]
Thank you.

Maybe all these questions of mine will help me or someone else out in the future when writing documentation. :-)
